src/styles/variables.js

```diff
@@ -84,3 +84,7 @@ export const overlayZIndex = '190'; // overlay-z-index
 export const tooltipZIndex = '190'; // tooltip-z-index
 export const bdlLineHeight = '20px'; // bdl-line-height
 export const bdlBorderRadiusSize = '4px'; // bdl-border-radius-size
+export const headerHeight = '56px'; // header-height
```

Comment:
I'm not sure if we want to add these globally, can we scope them to wherever they are used?